<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainAtlas</class>
 <widget class="QMainWindow" name="BrainAtlas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brain Atlas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;brain atlas name&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Label</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>z</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>left/right</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Notes</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="btnSelectAll">
            <property name="text">
             <string>Select all</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnAddBelow">
            <property name="text">
             <string>Add below</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnAddAbove">
            <property name="text">
             <string>Add above</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnClearSelection">
            <property name="text">
             <string>Clear selection</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="btnMoveToTop">
            <property name="text">
             <string>Move to top</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnMoveUp">
            <property name="text">
             <string>Move up</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="btnMoveDown">
            <property name="text">
             <string>Move down</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="btnMoveToBottom">
            <property name="text">
             <string>Move to bottom</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="GLViewWidget" name="graphicsView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkBox_5">
                 <property name="text">
                  <string>Show brain</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="2">
                <widget class="QPushButton" name="btnAxialDorsal">
                 <property name="text">
                  <string>Axial dorsal</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="btn3D">
                 <property name="text">
                  <string>3D</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="btnCoronalAnterior">
                 <property name="text">
                  <string>Coronal anterior</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btnSagittalLeft">
                 <property name="text">
                  <string>Sagittal left</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btnSagittalRight">
                 <property name="text">
                  <string>Sagittal right</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="btnAxialVentral">
                 <property name="text">
                  <string>Axial ventral</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="btnCoronalPosterior">
                 <property name="text">
                  <string>Coronal posterior</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBoxShowAxis">
               <property name="text">
                <string>Show axis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxShowGrid">
               <property name="text">
                <string>Show grid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxShowBrainRegions">
               <property name="text">
                <string>Show brain regions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxShowLabels">
               <property name="text">
                <string>Show labels</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_3"/>
    <addaction name="separator"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave_as_2"/>
    <addaction name="separator"/>
    <addaction name="actionImport_xml"/>
    <addaction name="actionImport_txt"/>
    <addaction name="actionImport_xls"/>
    <addaction name="actionExport_xml"/>
    <addaction name="actionExport_txt"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect_all"/>
    <addaction name="actionClear_selection"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionAdd_above"/>
    <addaction name="actionAdd_below"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionMove_up"/>
    <addaction name="actionMove_dowm"/>
    <addaction name="actionMove_to_top"/>
    <addaction name="actionMove_to_bottom"/>
   </widget>
   <widget class="QMenu" name="menuBrain_View">
    <property name="title">
     <string>Brain View</string>
    </property>
    <addaction name="actionGenerate_figure"/>
   </widget>
   <widget class="QMenu" name="menuSubject_Cohorts">
    <property name="title">
     <string>Subject Cohorts</string>
    </property>
    <addaction name="actionNew_MRI_Cohort"/>
    <addaction name="actionNew_fMRI_Cohort"/>
    <addaction name="actionNew_EEG_Cohort"/>
    <addaction name="actionNew_PET_Cohort"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About </string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBrain_View"/>
   <addaction name="menuSubject_Cohorts"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionOpen_3"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionHand"/>
   <addaction name="actionRotate"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actiontred"/>
   <addaction name="actionSagittal_left"/>
   <addaction name="actionSagittal_right"/>
   <addaction name="actionAxial_dorsal"/>
   <addaction name="actionAxial_ventral"/>
   <addaction name="actionCoronal_anterior"/>
   <addaction name="actionCoronal_posterior"/>
   <addaction name="separator"/>
   <addaction name="actionView_brain"/>
   <addaction name="actionShow_axis"/>
   <addaction name="actionShow_grid"/>
   <addaction name="actionShow_brain_regions"/>
   <addaction name="actionShow_labels"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionClear_selection">
   <property name="text">
    <string>Clear selection</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAdd_above">
   <property name="text">
    <string>Add above</string>
   </property>
  </action>
  <action name="actionAdd_below">
   <property name="text">
    <string>Add below</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionMove_up">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionMove_dowm">
   <property name="text">
    <string>Move dowm</string>
   </property>
  </action>
  <action name="actionMove_to_top">
   <property name="text">
    <string>Move to top</string>
   </property>
  </action>
  <action name="actionMove_to_bottom">
   <property name="text">
    <string>Move to bottom</string>
   </property>
  </action>
  <action name="actionGenerate_figure">
   <property name="text">
    <string>Generate figure</string>
   </property>
  </action>
  <action name="actionNew_MRI_Cohort">
   <property name="text">
    <string>New MRI Cohort ...</string>
   </property>
  </action>
  <action name="actionNew_fMRI_Cohort">
   <property name="text">
    <string>New fMRI Cohort ...</string>
   </property>
  </action>
  <action name="actionNew_EEG_Cohort">
   <property name="text">
    <string>New EEG Cohort ...</string>
   </property>
  </action>
  <action name="actionNew_PET_Cohort">
   <property name="text">
    <string>New PET Cohort ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionOpen_3">
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as_2">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionImport_xml">
   <property name="text">
    <string>Import (xml) ...</string>
   </property>
  </action>
  <action name="actionImport_txt">
   <property name="text">
    <string>Import (txt) ...</string>
   </property>
  </action>
  <action name="actionImport_xls">
   <property name="text">
    <string>Import (xls) ...</string>
   </property>
  </action>
  <action name="actionExport_xml">
   <property name="text">
    <string>Export (xml) ...</string>
   </property>
  </action>
  <action name="actionExport_txt">
   <property name="text">
    <string>Export (txt) ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionHand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hand</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionSagittal_left">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_sl.png</normaloff>:/buttons/icon_view_sl.png</iconset>
   </property>
   <property name="text">
    <string>Sagittal left</string>
   </property>
  </action>
  <action name="actionSagittal_right">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_sr.png</normaloff>:/buttons/icon_view_sr.png</iconset>
   </property>
   <property name="text">
    <string>Sagittal right</string>
   </property>
  </action>
  <action name="actionAxial_dorsal">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_ad.png</normaloff>:/buttons/icon_view_ad.png</iconset>
   </property>
   <property name="text">
    <string>Axial dorsal</string>
   </property>
  </action>
  <action name="actionAxial_ventral">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_av.png</normaloff>:/buttons/icon_view_av.png</iconset>
   </property>
   <property name="text">
    <string>Axial ventral</string>
   </property>
  </action>
  <action name="actionCoronal_anterior">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_ca.png</normaloff>:/buttons/icon_view_ca.png</iconset>
   </property>
   <property name="text">
    <string>Coronal anterior</string>
   </property>
  </action>
  <action name="actionCoronal_posterior">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_cp.png</normaloff>:/buttons/icon_view_cp.png</iconset>
   </property>
   <property name="text">
    <string>Coronal posterior</string>
   </property>
  </action>
  <action name="actionView_brain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_brain.png</normaloff>:/buttons/icon_brain.png</iconset>
   </property>
   <property name="text">
    <string>Show brain</string>
   </property>
  </action>
  <action name="actionShow_axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_axis.png</normaloff>:/buttons/icon_axis.png</iconset>
   </property>
   <property name="text">
    <string>Show axis</string>
   </property>
  </action>
  <action name="actionShow_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_grid.png</normaloff>:/buttons/icon_grid.png</iconset>
   </property>
   <property name="text">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionShow_brain_regions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_br.png</normaloff>:/buttons/icon_br.png</iconset>
   </property>
   <property name="text">
    <string>Show brain regions</string>
   </property>
  </action>
  <action name="actionShow_labels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_label.png</normaloff>:/buttons/icon_label.png</iconset>
   </property>
   <property name="text">
    <string>Show labels</string>
   </property>
  </action>
  <action name="actiontred">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/buttons/icon_view_3d.png</normaloff>:/buttons/icon_view_3d.png</iconset>
   </property>
   <property name="text">
    <string>tred</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.opengl</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
